project/src/main/java/com/google/sps/utilities/PrefilledInformationConstants.java

```diff
@@ -38,91 +38,32 @@
   public static final List<String> SKILLS =
       new ArrayList<>(
           Arrays.asList(
-              "american football",
```

Comment:
Minor thing: I am wondering if we also should update the current data to reflect this change since we might end up with a mix of these old skills and new skills.

Reply:
Yup, I can do this + will populate the db w more events as well